apps/baseai.dev/content/learn/learn/index.mdx

```diff
@@ -48,9 +48,10 @@ Create a directory in your local machine and navigate to it. Run the following c
 ```bash
 mkdir my-ai-project
 npm init -y
+npm install -S dotenv
```

Comment:
@ahmadbilaldev correct me if I am wrong but I believe we may not need `-S` now. Since the release of npm v5 in 2017, both `npm install` and `npm install -S` behave the same way.